File: src/Learn/Arrays101/sortedSquares.cs
public class Solution {
    public int[] SortedSquares(int[] A) {
    	int n = A.Length;
        int[] squares = new int[n];

        for(int i = 0; i < n; i++){
        	squares[i] = A[i] * A[i];
        }

        return Array.Sort(squares);
    }
}

File: src/Learn/Arrays101/sortedSquares.cpp
class Solution {
public:
    vector<int> sortedSquares(vector<int>& A) {
    	int N = A.size();
        vector<int> squares(N, 0);
        for(int i = 0; i < N; i++){
        	squares[i] = A[i] * A[i];
        }
        sort(squares.begin(), squares.end());
        return squares;
    }
};